### PR TITLE
Add empty properties to Microsoft.Genomics schema

### PR DESCRIPTION
**Pull request by jbagga**
Add empty properties to the schema

@tonybendis

### CODE REVIEW

**Comment by jbagga** [comment]
Ping @ravbhatnagar @Tiano2017 
Possible to review and merge soon? Thanks!

File: schemas/2017-08-01-preview/Microsoft.Genomics.json [hunk 1]
```diff
@@ -45,15 +45,28 @@
...
+            {
+              "type": "object",
+              "properties": {}
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Must exist in the request. Must not be null."
         }
```
Comment (TianoMS) [inline schemas/2017-08-01-preview/Microsoft.Genomics.json]: is this generated or hand-written? Is the swagger spec already updated?

**Comment by Tiano2017** [comment]
@jbagga PR looks good. Just one comment. Please take a look. Also, please sync the latest master.

File: schemas/2017-08-01-preview/Microsoft.Genomics.json (hunk 1 above)
Comment (jbagga) [reply schemas/2017-08-01-preview/Microsoft.Genomics.json]: @Tiano2017 hand-written. We do not have a swagger spec. Is it required? Can you point me to some documentation that shows how this schema is being used and how it relates to the swagger spec? 

Also this PR is an attempt/best guess to resolve the problem described here: https://github.com/Azure/azure-resource-manager-schemas/pull/529#issuecomment-440423147

Thanks!

**Comment by Tiano2017** [comment]
This is the swagger spec for Microsoft.Compute: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json. 

Each RP is supposed to have API review with their swagger spec, and schema file can be auto generated from swagger.

**Pull request merged**